amt/api/deps.py

```diff
@@ -69,7 +69,12 @@ def TemplateResponse(  # pyright: ignore [reportIncompatibleMethodOverride]
 
         if context is None:
             context = {}
-        context["csrftoken"] = request.state.csrftoken
+
+        if hasattr(request.state, "csrftoken"):
+            context["csrftoken"] = request.state.csrftoken
```

Comment:
Should we maybe make a ticket about this, I am doubting a bit whether this is a correct solution in the long run

Reply:
I prefer not because else my tests fail

Reply:
Sure, i will make a ticket out of it, but indeed understand you want to keep it here :)